pkg/web/templates/components/nav.html

```diff
@@ -32,7 +32,9 @@
     <ul tabindex="0" class="dropdown-content z-[1] bg-black text-white shadow menu menu-sm p-2 rounded-box w-52 font-semibold">
       <li><a href="/profile"><i class="fa-solid fa-user-gear"></i>User Profile</a></li>
       <li><a href="/about"><i class="fa-solid fa-gear"></i>Node Configuration</a></li>
+      {{ if .HasRole "Admin" }}
       <li><a href="/users"><i class="fa-solid fa-users"></i>Users Management</a></li>
+      {{ end }}
```

Comment:
Let's ask Edwin if we need to guard the whole user management page -- e.g. if it's fine if compliance and observers see the list of users on the node. 

Note that this will not prevent the user from going to `/users` directly, so if the answer to the above question is "no" then we should add an `authorize` to this route that allows only admins to access the page. If the answer is yes, then no backend change is needed.

Reply:
Is it possible to put this guard around the actions column in the users_management.html page? I realize probably why you opted to put this here was so you didn't have to change every table row with the clickable change password action, which probably has some javascript implications for the modal?

Reply:
It should be possible. I'll make the change.

Reply:
I should note that I tried return a bool instead of a string, but got an error preventing the condition from running on line 30 of the user_list.html` partial.